Streetcode/Streetcode.XUnitTest/MediatRTests/AdditionalContent/AdditionalContentTests/SubtitleTests/GetAllSubtitlesRequestHandlerTests.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System.Linq.Expressions;
+using AutoMapper;
+using Microsoft.EntityFrameworkCore.Query;
+using Moq;
+using Streetcode.BLL.DTO.AdditionalContent.Subtitles;
+using Streetcode.BLL.MediatR.AdditionalContent.Subtitle.GetAll;
+using Streetcode.DAL.Entities.AdditionalContent;
+using Streetcode.DAL.Enums;
+using Streetcode.DAL.Repositories.Interfaces.Base;
+using Xunit;
+
+namespace Streetcode.XUnitTest.MediatRTests.AdditionalContent.SubtitleTests
+{
+    public class GetAllSubtitlesRequestHandlerTests
+    {
+        private readonly Mock<IRepositoryWrapper> _mockRepo;
+        private readonly Mock<IMapper> _mockMapper;
+
+        public GetAllSubtitlesRequestHandlerTests()
+        {
+            _mockRepo = new Mock<IRepositoryWrapper>();
+            _mockMapper = new Mock<IMapper>();
+        }
+
+        private readonly List<Subtitle> subtitles = new List<Subtitle>
+        {
+            new Subtitle
+            {
+                Id = 1,
+                Status = SubtitleStatus.Editor,
+                StreetcodeId = 1
+            },
+            new Subtitle
+            {
+                Id = 2,
+                Status = SubtitleStatus.Illustrator,
+                StreetcodeId = 1
+            }
```

Comment:
same here, use a constant

Reply:
Even in GetAll? It will be used once, is it really mandatory?